internal/federationdomain/endpoints/auth/auth_handler_test.go

```diff
@@ -720,6 +724,16 @@ func TestAuthorizationEndpoint(t *testing.T) { //nolint:gocyclo
 			wantLocationHeader:                     expectedRedirectLocationForUpstreamOIDC(expectedUpstreamStateParam(nil, "", oidcUpstreamName, "oidc"), nil),
 			wantUpstreamStateParamInLocationHeader: true,
 			wantBodyStringWithLocationInHref:       true,
+			wantAuditLogs: func(encodedStateParam string) []string {
```

Comment:
We should probably come up with a more generic and less verbose way of checking the audit logs

Reply:
This has been done. Resolving this conversation.